virttest/qemu_vm.py

```diff
@@ -2693,7 +2695,7 @@ def add_secure_guest_object(params):
                 "ats": params.get("virtio_dev_ats"),
                 "aer": params.get("virtio_dev_aer")}
             for key, value in properties_to_be_set.items():
-                if value and key in dev_properties:
+                if key not in device.params and value and key in dev_properties:
```

Comment:
@PaulYuuu  Do we need keep this one after introduce the customer `virtio_dev_filter`  ?

Reply:
It's not a must but better to keep it to avoid wrong operations.

Reply:
I prefer to use one way to define the device options, same with what we discussed before introduce the filter :) 
Request for a correct `virtio_dev_filter` as the solution to define device options as user need.

Reply:
Yes, this would be safer, but we cannot guarantee how users will use, because there 2 parameters that can be used together, but they should have primary and secondary order.

Reply:
okay, you convinced me. thanks!